src/app/containers/Ad/Amp/AdSlot.jsx

```diff
@@ -54,10 +71,14 @@ const slotConfigurations = {
   },
 };
 
-const AdSlot = ({ service, slotType }) => {
+export const getAssetType = pageType =>
```

Comment:
Just to double check these are the only asset types which currently have amp ads served?

Reply:
Just wanted to check because this function will always default to`idx`